templates/zerver/emails/missed_message.subject.txt

```diff
@@ -1,7 +1,7 @@
 {% if show_message_content %}
     {% if group_pm %} {% trans %}Group PMs with {{ huddle_display_name }}{% endtrans %}
     {% elif private_message %} {% trans %}PMs with {{ sender_str }}{% endtrans %}
-    {% elif stream_email_notify or mention %} #{{ stream_header }}
+    {% elif stream_email_notify or mention %} {% if topic_resolved %} [{% trans %}Resolved{% endtrans %}] {% endif %}#{{ stream_header }}
```

Comment:
Glad to see you tagged this for translation! 

One thing that might be worth considering is whether it'd be better to have the translatable string be
"[Resolved] #{stream} > {topic}". Usually, we try to keep code/formatting out of translatable strings, since the format can be distracting, changes when we do non-functional changes, and can also be messed up if translators make mistakes (for example, cleverly replacing the brackets with parens would likely be bad; though I have no reason to believe someone would be tempted to do that). But in this case, I think there's an argument for including it here to better hint to the translator how this will appear, when perhaps deciding which form of "Resolved" to use. 

Thoughts? I guess if we wanted to change this, we'd want a little prep commit to refactor the existing string construction to replace the `stream_header` variable, etc.

Reply:
I think that's a good rationale. I don't "really" speak any languages other than English so I'm not super qualified to have a strong stance here, but I remember from what little French I studied that this type of verb would conjugate in a variety of ways depending on the traits of the noun it describes; I'd bet it matters in more languages than not.

It also, as you allude to, allows moving logic out of this file and back into `.py` files that can be type-checked and linted and so forth. I generally like such moves, so this is a good excuse to do that.

Reply:
Implemented in 5bcff13e20

Reply:
Yeah, I guess the main thing thing I had in mind is that some languages, like Arabic, are RTL, and I have no idea what the right way to translate this is for an RTL language, but the translator probably does, and it seems worth giving them to control to adjust the word order for the full subject line.